pkg/api/types.go

```diff
@@ -3593,13 +3593,7 @@ type EventList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // List holds a list of objects, which may not be known by the server.
-type List struct {
-	metav1.TypeMeta
-	// +optional
-	metav1.ListMeta
-
-	Items []runtime.Object
-}
+type List metainternalversion.List
```

Comment:
Sure. Actually I am surprised that this alias works, compare https://github.com/kubernetes/gengo/pull/73.